lib/active_model/serializer/attributes.rb

```diff
@@ -14,10 +14,12 @@ module Attributes
 
         # Return the +attributes+ of +object+ as presented
         # by the serializer.
-        def attributes(requested_attrs = nil, reload = false)
+        def attributes(options = {}, reload = false)
+          requested_attrs = options[:only]
+          excepts = Array(options[:except])
```

Comment:
@bf4 
active_model_serializers (0.10.0.rc4)
Say I have a `Chatroom` serializer: 

```
class ChatroomSerializer < ActiveModel::Serializer
  attributes :id, :name, :description, :room_type, :subscribed
  has_many :subscribers
  has_many :visitors

  def subscribed
    object.subscribers.include? current_user
  end
end
```
Now sometimes the client may only want basic attributes for the chatroom like `name`, `description`, `room_type`...etc. and doesn't care about the associations, so I'd like to do something like this in my actions:
`render json: @chatroom, except: [:subscribers, :visitors]`

or even something like this: 
`render json: @chatroom, except: params[:except_attrs]`

I know I can specify a specific serializer to render, but I don't want to create a separate serializer for each small variation, am I making sense?